pulp-glue/tests/conftest.py

```diff
@@ -0,0 +1,26 @@
+import typing as t
+
+import pytest
+
+from pulp_glue.common.context import PulpContext
+from pulp_glue.common.openapi import BasicAuthProvider
+
+
+class PulpTestContext(PulpContext):
+    # TODO check if we can just make the base class ignore echo.
```

Comment:
What does this mean, doesn't the base class raise a `NotImplemented` error?

Reply:
It does, and that means, the PulpContext is unusable as is. I think that is a bad thing.